sretoolbox/container/image.py

```diff
@@ -472,7 +473,21 @@ def __eq__(self, other):
         if manifest_version == 1:
             layers_key = 'fsLayers'
         else:
-            layers_key = 'layers'
+            manifest_media_type = manifest['mediaType']
+            other_manifest_media_type = other_manifest['mediaType']
+
+            if manifest_media_type != other_manifest_media_type:
```

Comment:
agreed... __eq__ will aways have a meaning and in this case it has the correct meaning: images are not the same.

Thinking about the extended scope, when a simple image can be in the manifest on the multi-image, the interface I would expect is something like `simple_image in multi_image.manifest`, but yeah, that's a different topic :)

Reply:
That interface could work for me as well ;-)